doc/doc-site/docs/overview/orders.md

```diff
@@ -278,323 +278,128 @@ python3
->>> lot_size
-100000
->>> tick_size = get_tick_size_integer(tick_size_decimal, quote_decimals)
->>> tick_size
-1
->>> min_size_float = 5/17792.27
->>> min_size_decimal = f"{min_size_float}"
->>> min_size = get_min_size_integer(min_size_decimal, base_decimals, lot_size)
->>> min_size
-3
-```
-
-Therefore our final market parameters are as follows:
-
-| Parameter    | Units                | Value  | Meaning                  |
-| ------------ | -------------------- | ------ | ------------------------ |
-| Lot size     | Subunits (of `WBTC`) | 100000 | Order size granularity   |
-| Tick size    | Subunits (of `USDC`) | 1      | Lot value granularity    |
-| Minimum size | Lots (of `WBTC`)     | 3      | Minimum limit order lots |
-
-______________________________________________________________________
-
-Next, consider a liquid staking derivative `sAPT` trading against `APT`, `sAPT/APT`, both having 8 decimals.
-Since the two assets have almost the same nominal price, price granularity will need to be higher here:
-
-| Variable          | Value          |
-| ----------------- | -------------- |
-| Decimal lot size  | 0.01 `sAPT`    |
-| Decimal tick size | 0.000001 `APT` |
-| Base decimals     | 8              |
-| Quote decimals    | 8              |
-
-Let's check the minimum and maximum price here using the SDK:
-
-```python
->>> lot_size = "0.01"
->>> tick_size = "0.000001"
->>> get_min_quote_per_base_nominal(lot_size, tick_size)
-0.0001
->>> get_max_quote_per_base_nominal(lot_size, tick_size)
-429496.7295
-```
-
-This gives us a price precision of 1/100th of a penny, with room to go even more precise should we desire.
+This gives us a price precision of 1/10000th of a penny, with no room to go more precise should we desire.
 However we recall that the most granular possible price isn't necessarily desirable due to almost every order then occupying its own price level, cluttering user interfaces.
-Some coarseness is desirable for that reason.
+Some coarseness is desirable for that reason, therefore the market creator might instead use less price granularity and choose to preserve size precision.
 Although it is possible for user interfaces to summarize orders into price levels that don't actually exist, this approach would represent to the user information that is not accurate, and is not recommended.
-Should we use the maximum precision of 0.00000001 `APT` then we observe that the maximum price lowers but remains high enough for this pair, where the assets have almost the same value:
 
-```python
->>> lot_size = "0.01"
->>> tick_size = "0.00000001"
->>> get_min_quote_per_base_nominal(lot_size, tick_size)
-1e-06 # 0.000001
->>> get_max_quote_per_base_nominal(lot_size, tick_size)
-4294.967295
-```
-
-Last, let's calculate market parameters again using \$5 as our minimum order size:
+Let's check the maximum price of this market, keeping in mind that it should revert to 1 since the assets have equal value:
```

Comment:
How about?

"... that it should hover around 1 since the assets have almost the same value"

Reply:
done